sprocs/variants_select_for_assessment_instance_grading.sql

```diff
@@ -18,7 +18,7 @@ BEGIN
         JOIN instance_questions AS iq ON (iq.id = v.instance_question_id)
         JOIN assessment_instances AS ai ON (ai.id = iq.assessment_instance_id)
         JOIN questions AS q ON (q.id = v.question_id)
-        JOIN pl_courses AS c ON (c.id = q.course_id)
+        JOIN pl_courses AS c ON (c.id = v.course_id)
```

Comment:
Ooooh, this is a tricky one. We need `course` to correspond to the question's course so that we know where to find it on disk, but we also add the entire `course` object to any reported issues. I think we might need to return both `question_course` and `variant_course` here, and pipe both of them around?

Reply:
Nice catch.

Reply:
See edits.